src/computeFontSize.ts

```diff
@@ -0,0 +1,63 @@
+import * as vscode from "vscode";
+
+export interface FontSize {
+  fontSize: number;
+  fontWidth: number;
+  fontHeight: number;
+}
+
+export default async function (): Promise<FontSize> {
+  const fontRatios = await getFontRatios();
+  const fontSize = getFontSize();
+  return {
+    fontSize,
+    fontWidth: fontSize * fontRatios[0],
+    fontHeight: fontSize * fontRatios[1],
```

Comment:
Cool you get the height now; that's nice

Reply:
Yeah did that at the same time.